bin/crew

```diff
@@ -1274,14 +1309,8 @@ def install
   end
 
   # add to installed packages
-  @device[:installed_packages].push(name: @pkg.name, version: @pkg.version, sha256: PackageUtils.get_sha256(@pkg, build_from_source: @opt_source))
-  File.open("#{CREW_CONFIG_PATH}/device.json.tmp", 'w') do |file|
-    output = JSON.parse @device.to_json
-    file.write JSON.pretty_generate(output)
-  end
-  # Copy over original if the write to the tmp file succeeds.
-  FileUtils.cp "#{CREW_CONFIG_PATH}/device.json.tmp", File.join(CREW_CONFIG_PATH, 'device.json')
-  FileUtils.rm "#{CREW_CONFIG_PATH}/device.json.tmp"
+  @device_json['installed_packages'].push(name: @pkg.name, version: @pkg.version, sha256: PackageUtils.get_sha256(@pkg, build_from_source: @opt_source))
```

Comment:
Check if package is already in installed_packages prior to push?

Reply:
If the package is already installed we still need to delete the package info... We could do a check for that, if that would make more sense?

Reply:
I've added a package installed check right before it is added.  (Right now we DO have a situation where the package keeps getting added...)